packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -18,14 +18,18 @@ import {WepublishEditor} from './editor/wepublishEditor'
 
 export interface RichTextBlockProps extends BlockProps<RichTextBlockValue> {
   displayOnly?: boolean
+  showCharCount?: boolean
+  getStringText?: (text: string) => void
```

Comment:
Not needed.

Reply:
Why?

Reply:
because we don't need the functionality in the block.